packages/imperative/src/doc/IImperativeConfig.ts

```diff
@@ -326,4 +326,12 @@ export interface IImperativeConfig {
      * @memberof IImperativeConfig
      */
     apimlConnLookup?: IApimlSvcAttrs[];
+
+    /**
+     * If Imperative should run in Daemon mode
+     * This should only be specified for CLIs
+     * @type {boolean}
+     * @memberof IImperativeConfig
+     */
+    daemonMode?: boolean;
```

Comment:
the description of this is that it "should" run in daemon mode, but presumably it is possible to not run in daemon mode.  is that correct?  will zowe cli set this in a PR for that repo?

Reply:
As stated in my previous comment, this option allows Imperative to initialize while suppressing configuration errors in Daemon mode. It should not be true when daemon mode is not running, because the configuration is never reloaded outside of when the --dcd option is specified (which is only true in Daemon mode).

Reply:
ok thanks, if `--daemon` is found at startup, `daemonMode` will be set to `true` - is that the correct summary?

Reply:
That is correct.